com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -557,6 +559,11 @@ public float cascade2Split
             get { return m_Cascade2Split; }
         }
 
+        public Vector2 cascade3Split
```

Comment:
we should add API docs for all new public API (also take opportunity to add to existing related missing one)

Reply:
The API docs should in theory be exactly as the one for HDRP for similar things. I will look into that.